EventsExpress/ClientApp/src/components/RegistrationForm/TellUsMoreAboutYourself.js

```diff
@@ -0,0 +1,150 @@
+﻿import "./RegistrationForm.css";
+import React, { useState } from "react";
+import { Grid, Button } from "@material-ui/core";
+import { reduxForm, Field, getFormValues } from "redux-form";
+import parentStatusEnum from '../../constants/parentStatusEnum';
+import relationShipStatusEnum from '../../constants/relationShipStatusEnum';
+import theTypeOfLeisureEnum from '../../constants/theTypeOfLeisureEnum';
+import reasonsForUsingTheSiteEnum from '../../constants/reasonsForUsingTheSiteEnum';
+import eventTypeEnum from '../../constants/eventTypeEnum';
+import { renderCheckbox, radioButton } from '../helpers/form-helpers';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Radio from '@material-ui/core/Radio';
+
+const TellUsMoreAbotYourself = (props) => {
+    const { handleSubmit, previousPage } = props;
+
+    return (
+        <div className="Step4">
+            <form onSubmit={handleSubmit}>
+            <h1 className="text-left">Step4: Tell us more about yourself. </h1>
+            <Grid container spacing={2} className="text-left">
+                <Grid item xs={4}>
+                    <div class="stepper-tellusmore-block">
+                    <label className="font-weight-bold">Select parent status</label>
+                        <Field name="parentstatus" component={radioButton} parse={String} >
```

Comment:
Fix formatting, please, so would be a bit readable

Also, could we pass label text into Field and handle rendering there?